.github/workflows/publish-to-other-than-github.yaml

```diff
@@ -19,6 +19,60 @@ on:
     types: [ocm-cli-release]
 
 jobs:
+  push-to-brew-tap:
+    name: Update Homebrew Tap
+    runs-on: macos-latest
+    env:
+      REPO: open-component-model/homebrew-tap
+    steps:
+    - name: Ensure proper version
+      run: |
+        if [ -n "${{ github.event.inputs.version }}" ]; then
+          echo "RELEASE_VERSION=$(echo ${{ github.event.inputs.version }} | tr -d ['v'])" >> $GITHUB_ENV
+          exit 0
+        fi
+        if [ -n "${{ github.event.client_payload.version }}" ]; then
+          echo "RELEASE_VERSION=$(echo ${{ github.event.client_payload.version }} | tr -d ['v'])" >> $GITHUB_ENV
+          exit 0
+        fi
+        echo "Version not provided"
+        exit 1
+    - name: Generate token
+      id: generate_token
+      uses: tibdex/github-app-token@v2
+      with:
+        app_id: ${{ secrets.OCMBOT_APP_ID }}
+        private_key: ${{ secrets.OCMBOT_PRIV_KEY }}
+    - name: Get Update Script
+      uses: actions/checkout@v4
+      with:
+        path: scripts
+        sparse-checkout: 'hack/brew'
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        repository: ${{ env.REPO }}
+        token: ${{ steps.generate_token.outputs.token }}
+    - name: Update Homebrew Tap
+      run: |
+        formula=$(go run scripts/hack/brew \
+            --version ${{ env.RELEASE_VERSION }} \
+            --template scripts/hack/brew/ocm_formula_template.rb.tpl \
+            --outputDirectory Formula)
+        mkdir -p Aliases
+        ln -s Formula/$(basename formula) Aliases/ocm
+        rm -r scripts
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v7
```

Comment:
öhmm... maybe stupid question, but where is the PR actually created? Automatically on the latest checked-out repo? Or always on the one, where the action runs?

Reply:
PR is created on the Tap Repository.

This is an example PR created from an internal testing repository: https://github.com/open-component-model/homebrew-tap/pull/3
The testing run (only accessible by org members): https://github.com/open-component-model/ocm-cicd-playground/actions/runs/11748205455/job/32731771674

a working tap repository can be found in https://github.com/jakobmoellerdev/homebrew-tap that follows this method and can be used for testing installations like

```
brew install jakobmoellerdev/tap/ocm # redirects via the alias to always the latest published
brew install jakobmoellerdev/tap/ocm@0.17.0 # versioned publish
```